app/controllers/chargeback_assignment_controller.rb

```diff
@@ -4,108 +4,68 @@ class ChargebackAssignmentController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
-  include Mixins::SavedReportPaging
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
     @table_name ||= "chargeback_assignment"
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
-  end
-
-  def explorer
+  def index
```

Comment:
I'm pretty sure this brings in some inconsistency, just look at how `index` is defined in other controllers. Can you please explain why is this necessary?

Reply:
@skateman since this screen does not really have a list just a form with a type drop down, I didn't name this show_list, so we need something for landing page, what do you recommend it should be?